doc/release/v3_1_1.md

```diff
@@ -49,6 +49,8 @@ Bug Fixes
 * Made optional the view of `IFrameGrabberControls` in `RGBDSensorWrapper`
   (#1875).
 * Fixed header inclusion in `ImplementControlLimits2.h`.
+* Updated `AnalogWrapper` to open multiple ros topics for `wrenchStamped`
```

Comment:
This PR targets `devel` therefore this should be in `doc/release/v3_2_0.md`

Reply:
Done